components/bower_components/system.js/.bower.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Actually, is there any good reason to store libraries, provided by bower in the Git?
I've added new libraries to Git only because the old were stored there.